dotnet-grad/Models/TestContext.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+
+namespace dotnet_grad.Models
+{
+  public class TestContext : DbContext
+  {
+
+    public DbSet<UserModel> Users { get; set; }
+    public DbSet<OrganisationModel> Organisations { get; set; }
+    public DbSet<AddressModel> Addresses { get; set; }
+
+    public string DbPath { get; }
+    public TestContext()
+    {
+      var folder = Environment.SpecialFolder.LocalApplicationData;
+      var path = Environment.GetFolderPath(folder);
+      DbPath = System.IO.Path.Join(path, "testing.db");
+    }
+    protected override void OnConfiguring(DbContextOptionsBuilder options)
```

Comment:
As discussed during the demo session, it is generally better to have your connection string for your database (and anything that can and will change based on the environment) be defined within the configuration file and consumed as such. IE in this case you'd define your entire connection string in `appsettings.json` under `"ConnectionStrings": { "somekey" : "Data Source ..."` and inject IConfiguration and consume it as such `_configiration.GetConnectionString("somekey")` etc.